core/services/relay/evm/tron_transmitter.go

```diff
@@ -0,0 +1,171 @@
+	if err != nil {
+		return nil, err
+	}
+
+	// Construct the Tron contract transmitter, it's slightly different from the EVM contract transmitter and due to mismatching types we have to apply the transmitter options manually
+	transmitter := tron.NewOCRContractTransmitter(ctx, opts.TransmissionsCache, tronsdk.EVMAddressToAddress(opts.ConfigWatcher.contractAddress), tronsdk.EVMAddressToAddress(senderAddress), tronTXM, opts.Logger).WithEthereumKeystore()
+	transmitterOptions := &transmitterOps{
+		reportToEvmTxMeta: nil,
+		excludeSigs:       false,
+		retention:         0,
+		maxLogsKept:       0,
+	}
+
+	for _, opt := range opts.OCRTransmitterOpts {
+		opt(transmitterOptions)
+	}
+
+	if transmitterOptions.excludeSigs {
+		opts.Logger.Info("Excluding signatures from transmissions")
+		transmitter = transmitter.WithExcludeSignatures()
+	}
+	if transmitterOptions.reportToEvmTxMeta != nil {
+		opts.Logger.Info("Using EVM TxMeta for Tron Transmissions")
+		transmitter = transmitter.WithReportToEthMetadata(transmitterOptions.reportToEvmTxMeta)
+	}
+
+	tronTransmitter := newTronTransmitterWrapper(transmitter, tronTXM)
+
+	return tronTransmitter, nil
+}
+
+var _ ContractTransmitter = (*tronTransmitterWrapper)(nil)
+
+// Simple wrapper around the tron.ContractTransmitter to provide start / stop hooks to the tron txm
+type tronTransmitterWrapper struct {
+	transmitter tron.ContractTransmitter
+	txm         *trontxm.TronTxm
+}
+
+func newTronTransmitterWrapper(transmitter tron.ContractTransmitter, txm *trontxm.TronTxm) ContractTransmitter {
+	return &tronTransmitterWrapper{
+		transmitter: transmitter,
+		txm:         txm,
+	}
+}
+
+// The Tron Transmitter doesn't close the txm, so we'll close it here
+// NOTE: This is called by the Close() method of either the CommitProvider or the ExecProvider
+func (t *tronTransmitterWrapper) Close() error {
+	err := t.txm.Close()
+	if err != nil {
+		return err
+	}
+
+	return t.transmitter.Close()
+}
+
+func (t *tronTransmitterWrapper) FromAccount(ctx context.Context) (types.Account, error) {
+	return t.transmitter.FromAccount(ctx)
+}
+
+func (t *tronTransmitterWrapper) HealthReport() map[string]error {
+	return t.transmitter.HealthReport()
+}
+
+func (t *tronTransmitterWrapper) LatestConfigDigestAndEpoch(ctx context.Context) (types.ConfigDigest, uint32, error) {
+	return t.transmitter.LatestConfigDigestAndEpoch(ctx)
+}
+
+func (t *tronTransmitterWrapper) Name() string {
+	return t.transmitter.Name()
+}
+
+func (t *tronTransmitterWrapper) Ready() error {
+	return t.transmitter.Ready()
+}
+
+// As the Tron Transmitter doesn't start the txm, we need to start it within the start hook
+func (t *tronTransmitterWrapper) Start(ctx context.Context) error {
+	t.txm.Logger.Info("Starting Tron TXM")
+
+	// NOTE: The txm needs to be started before the contract transmitter, so we check if it's ready and start it if it's not already started
+	if err := t.txm.Start(ctx); err != nil {
```

Comment:
As discussed offline, let's drop this `tronTransmitterWrapper` component and move TRON TXM init and lifecycle to the chain component, and expose it via existing `ChainTronSupport` type.